src/app/features/auth/auth/auth.component.scss

```diff
@@ -0,0 +1,64 @@
+@use './src/styles' as st;
+
+$icon: st.$accent-gold-500;
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+.login-view-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  padding: clamp(21px, 3.5vw, 120px) clamp(25px, 10.5vw, 160px);
+  min-width: clamp(250px, 50vw, 600px);
+
+  .text-body-big {
+    font-family: Eczar;
+  }
```

Comment:
Jeśli z font-family coś nie działało, to wydaje mi się, że powinno być zmienione w stylach globalnych a nie tylko lokalnie.

Reply:
Ok, zgodnie z naszą dyskusją zmieniam styl font-family na domyślny jakim jest Mukta Vaani i zaktualizuje makietę, na której był Eczar.